locales/base/translation.json

```diff
@@ -27,6 +27,15 @@
     "body": "To switch Data Saver on and off repeatedly, you will need to restart the app and return to this screen.",
     "restart": "Restart to Switch"
   },
+  "revokePhoneNumber": {
+    "title": "Unlink Phone Number",
```

Comment:
yes i discussed this with Reed - wording is hard, "revoke" was not used here because it sounds technical and user unfriendly. we did consider "disconnect" because of the "connect" during onboarding but disconnecting a phone number makes it sound like you're physically disabling your phone plan :/ we ended up going with "unlink" but i'd be happy to discuss again. did you have any options in mind?